packages/core/src/Cardano/types/ProtocolParameters.ts

```diff
@@ -95,7 +101,19 @@ type NewProtocolParamsInBabbage = {
   coinsPerUtxoByte: number;
 };
 
+// coinsPerUtxoWord was replaced by coinsPerUtxoByte and extraEntropy was deprecated.
 type BabbageProtocolParameters = Omit<AlonzoProtocolParams, 'coinsPerUtxoWord' | 'extraEntropy'> &
   NewProtocolParamsInBabbage;
 
 export type ProtocolParameters = BabbageProtocolParameters;
+
+// Even tho extraEntropy was deprecated on babbage era, it is still present in the ProtocolParametersUpdate structure
+// since this structure is backward compatible with all eras.
+export type ProtocolParametersUpdate = Partial<ProtocolParameters & Pick<AlonzoProtocolParams, 'extraEntropy'>>;
+
+export type ProposedProtocolParameterUpdates = Map<Crypto.Hash28ByteBase16, ProtocolParametersUpdate>;
```

Comment:
What is this hash of? :thinking:

Reply:
There's only 1 genesis block though, so it would always be a map of size 1? If that's the case, then maybe it's better to model as `{hash, update}`?